ja2r.gemspec

```diff
@@ -9,11 +9,12 @@ Gem::Specification.new do |s|
   s.homepage    = 'https://github.com/mkon/ja2r'
   s.summary     = 'Simple JSON-API to ruby object conversion'
   s.license     = 'MIT'
-  s.required_ruby_version = '>= 3.1', '< 3.4'
+  s.required_ruby_version = '>= 3.1', '< 3.5'
 
   s.files = Dir['{app,config,db,lib}/**/*', 'LICENSE', 'README.md']
 
   s.add_dependency 'activesupport', '>= 6.1', '< 8.0'
+  s.add_dependency 'logger', '~> 1.6'
```

Comment:
Do we need logger as dependency? I did not find any use of logger. I removed it and tests worked just fine.

Reply:
Doesn't look like it. I've pushed a commit to remove it.

Not sure what I was doing yesterday :)